basic-application/basic-application-core/src/main/java/fr/openwide/core/basicapp/core/business/referential/model/search/CitySort.java

```diff
@@ -0,0 +1,53 @@
+package fr.openwide.core.basicapp.core.business.referential.model.search;
+
+import java.util.List;
+
+import org.apache.lucene.search.SortField;
+
+import com.google.common.collect.ImmutableList;
+
+import fr.openwide.core.basicapp.core.business.referential.model.City;
+import fr.openwide.core.jpa.business.generic.model.GenericEntity;
+import fr.openwide.core.jpa.more.business.sort.ISort;
+import fr.openwide.core.jpa.more.business.sort.SortUtils;
+
+public enum CitySort implements ISort<SortField> {
```

Comment:
This could totally be renamed GenericListItemSort and pushed up to owsi-core-components-jpa-more in package `fr.openwide.core.jpa.more.business.generic.model.search`.